src/stories/Library/hero/Hero.tsx

```diff
@@ -22,13 +22,13 @@ const Hero: React.FunctionComponent<HeroProps> = ({
     <section className="hero">
       <a href="/" className="hero-content arrow arrow__hover--right-large">
         <TagCategories tags={["Arrangement"]} />
-        <div className="hero-date">
+        <div className="hero__date">
           {contentType && (
             <>
-              <span className="hero-date__text">{contentType}</span>|
+              <span className="hero__date__text">{contentType}</span>|
             </>
           )}
-          <time className="hero-date__date">{date}</time>
+          <time className="hero__date__date">{date}</time>
```

Comment:
Sorry is should've been more discriptive on first view. 

What i ment was, that if the only "date" reference here, is `hero__date__date`, then maybe `hero__date`should be named differently. maybe something like hero-content-info or hero-content-details with __type and __date for the span/time elements. 

I'll leave that up to you to decide.